lib/ansible/modules/system/sysctl.py

```diff
@@ -36,6 +36,12 @@
             - Whether the entry should be present or absent in the sysctl file.
         choices: [ "present", "absent" ]
         default: present
+    no_extra_spaces:
```

Comment:
Maybe rebrand this to `with_extra_spaces`? Otherwise it's double negation by default `no_extra_spaces: no`, which is confusing.

Reply:
Although I'd want the arg name to represent what it does better. When I just look at the name I wonder: spaces? extra? why? where? trailing? starting? empty line separator? what does it all mean?
I believe it should be clear from just looking at the name what it does without having to look up the docs.
Maybe `space_wrap_assignment`? Any better ideas?

Reply:
@webknjaz Thank you for the feedback. Totally agree, especially with the 2nd point. I also have another PR regarding the `ignorerrors` parameter that I renamed for exactly the same reason. I will think about it and see if I find a better fitting name.

Reply:
I saw the same problem.  However we constrained because "no_extra_spaces" is already in use (ini_file).  If we go with a better name here, we should deprecate no_extra_spaces in in_file and replace it with the better name as well.

Reply:
Ok, also a good point. Everything that came to my mind has super long names like `wrap_assignment_operator_in_spaces` or `space_wrap_assignment_operator` for example. These names are long but at the same time the name says exactly what this parameter is doing.

Reply:
second one sounds better to me

Reply:
Renamed it to `space_wrap_assignment_operator` and updated the fragment and tests.